rs/pocket_ic_server/src/pocket_ic.rs

```diff
@@ -3345,6 +3348,7 @@ impl Operation for QueryRequest {
                     Arc::new(StandaloneIngressSigVerifier),
                     NNSDelegationReader::new(delegation_rx, subnet.replica_logger.clone()),
                     query_handler,
+                    query::Version::V2,
```

Comment:
Do we need any TODOs here and in other places to switch to the new version later on?